source/components/text-input/index.html

```diff
@@ -5,158 +5,103 @@
   <div class="demo">
-    <form>
-      <div class="hxFormGroup">
-        <label for="txtEmail">Default</label>
-        <input type="email" id="txtEmail" placeholder="Email">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtInvalidEmail">Invalid</label>
-        <input type="email" class="-hxInvalid" id="txtInvalidEmail" placeholder="Email">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtDisabledEmail">Disabled</label>
-        <input type="email" id="txtDisabledEmail" placeholder="Email" disabled>
-      </div>
-    </form>
-  </div>
-  {% code 'html' %}
-    <form>
-      <div class="hxFormGroup">
-        <label for="txtEmail">Default</label>
-        <input type="email" id="txtEmail" placeholder="Email">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtInvalidEmail">Invalid</label>
-        <input type="email" class="-hxInvalid" id="txtInvalidEmail" placeholder="Email">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtDisabledEmail">Disabled</label>
-        <input type="email" id="txtDisabledEmail" placeholder="Email" disabled>
-      </div>
-    </form>
-  {% endcode %}
-</section>
+    <input
+      class="hxTextCtrl"
+      placeholder="Default Text Input"
+      type="text" />
+    <br />
 
-<section>
-  <h2 class="hxSectionTitle" id="password">Password</h2>
-  <div class="demo">
-    <form>
-      <div class="hxFormGroup">
-        <label for="txtPassword">Password</label>
-        <input type="password" id="txtPassword" placeholder="Password">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtInvalidPassword">Invalid</label>
-        <input type="password" class="-hxInvalid" id="txtInvalidPassword" placeholder="Password">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtDisabledPassword">Disabled</label>
-        <input type="password" id="txtDisabledPassword" placeholder="Password" disabled>
-      </div>
-    </form>
+    <input
+      class="hxTextCtrl"
+      invalid
+      placeholder="Invalid Text Input"
+      type="text" />
+    <br />
+
+    <input
+      class="hxTextCtrl"
+      disabled
+      placeholder="Disabled Text Input"
+      type="text" />
   </div>
   {% code 'html' %}
-    <form>
-      <div class="hxFormGroup">
-        <label for="txtPassword">Password</label>
-        <input type="password" id="txtPassword" placeholder="Password">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtInvalidPassword">Invalid</label>
-        <input type="password" class="-hxInvalid" id="txtInvalidPassword" placeholder="Password">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtDisabledPassword">Disabled</label>
-        <input type="password" id="txtDisabledPassword" placeholder="Password" disabled>
-      </div>
-    </form>
+    <input
+      class="hxTextCtrl"
+      placeholder="Default Text Input"
+      type="text" />
+    <br />
```

Comment:
remove `<br />` from code snippets

Reply:
Removed in snippets. This comment is on the demo :)